azext_iot/central/commands_device.py

```diff
@@ -88,6 +90,36 @@ def run_command(
         payload=payload,
     )
 
+def run_manual_failover(
+    cmd,
+    app_id: str,
+    device_id: str,
+    ttl_minutes=None,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+):
+    if ttl_minutes and ttl_minutes < 1:
```

Comment:
we should have this validation in the `params` layer not in the `controller` layer